introduction.md

```diff
@@ -1,3 +1,9 @@
 I'm Phitherek_ and I'm introducing myself.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?